content/product-engineering/planning.md

```diff
@@ -0,0 +1,214 @@
+# Product and engineering planning
+
+- [Product and engineering planning](#product-and-engineering-planning)
+  - [Philosophy](#philosophy)
+  - [Definitions](#definitions)
+  - [Annual planning process](#annual-planning-process)
+  - [Quarterly planning process](#quarterly-planning-process)
+  - [Sharing progress](#sharing-progress)
+    - [Status definitions](#status-definitions)
+    - [GitHub project tracker](#github-project-tracker)
+    - [OKR slides](#okr-slides)
+    - [Weekly team updates](#weekly-team-updates)
+    - [Monthly team updates](#monthly-team-updates)
+  - [Quarterly retrospectives](#quarterly-retrospectives)
+
+## Philosophy
+
+Our planning process defines how we create goals and track progress toward those goals.
+
+Properties of a good plan:
+
+- The plan addresses the most important business needs (and is subject to change if the business needs change).
+- The plan is ambitious and feasible.
+- The plan provides a clear direction about what success looks like so the teams have space to figure out how to achieve success and so teams understand what is not a priority.
+
+To acheive a good plan, our planning process based on the following principles:
+
+- Everyone on our team has context and insights that are valuable to inputs into the planning process.
+- success we want to understand and be able to measure the impact and value the work we commit to will have on customers, our internal teams, and the overall success of Sourcegraph.
+- Arriving at a good plan is an iterative process that is neither exclusively top-down nor exclusively bottom-up because there is a cyclical information dependency. Company level strategy is informed by what is working and not working about our business (context our teams have), and team plans are informed by what our company level strategy is (a decision ultimately owned by our CEO).
+
+TODO(nick): write more here
+
+## Definitions
+
+- Strategy: Our [strategy](strategy.md) is a concise description of how product and engineering contribute to the top-level company objectives. It sets a clear direction so we stay focused on what is important and have appropriate trade-off discussions if necessary.
+- OKRs: [OKRs](../company/goals/guidelines.md) define how we measure success.
+- Roadmap items: The concrete high-level things that we are delivering to achieve success (as measured by OKRs).
```

Comment:
Maybe add milestones as a fourth definition here because we're now tracking those as separate entities?